runners/flink/flink_runner.gradle

```diff
@@ -152,6 +152,7 @@ dependencies {
   compile "org.apache.flink:flink-metrics-core:$flink_version"
   compile "org.apache.flink:flink-java:$flink_version"
   compile "org.apache.flink:flink-runtime_2.11:$flink_version"
+  compile("com.lyft:streamingplatform-events-source:2.1-dryft-SNAPSHOT")
```

Comment:
This should be further down in the lyft specific block.

Reply:
Also, the size of the job server has grown significantly with this addition. Have you checked that we are only pulling in the minimum transitive dependencies?